docs/Guide/application-commands/application-command-registry/registering-context-commands.mdx

```diff
@@ -44,47 +43,38 @@ export class SlashCommand extends Command {
   }
 
   public override contextMenuRun(interaction: Command.ContextMenuInteraction) {
-    // Use `isMessageContextMenu` to ensure this command was executed as a Message Context Menu Command.
-    if (interaction.isMessageContextMenu()) {
-    }
```

Comment:
revert

Reply:
I added the comment back in the last section that covers implementing `contextMenuRun`. The previous section on Chat Input Commands has `chatInputRun` empty until the last section, so I'd like to match the format as well.